src/core/Akka.Streams.TestKit.Tests/BaseTwoStreamsSetup.cs

```diff
@@ -20,7 +20,7 @@ public abstract class BaseTwoStreamsSetup<TOutputs> : AkkaSpec
     {
         protected readonly ActorMaterializer Materializer;
 
-        public BaseTwoStreamsSetup(ITestOutputHelper output = null) : base(output)
+        protected BaseTwoStreamsSetup(ITestOutputHelper output = null) : base(output)
```

Comment:
We have to add APISpec for Streams, because we could have public API breaking changes